tests/tck/conftest.py

```diff
@@ -240,6 +240,26 @@ def clone_space(graph_spaces, session, request):
     graph_spaces["space_desc"] = space_desc
     graph_spaces["drop_space"] = True
 
+
+@when(parse("try to execute query on {space} from user {username} with password {password}:\n{query}"))
+def execute_query_from_user(query, space, username, password, conn_pool_to_second_graph_service, graph_spaces, session, request):
```

Comment:
sorry for this late fix:
- session() now return a dict, and default use session['current'], so that sessions could be changed, and session['current'] default use user and password from config to init user.
- change user now will work for next sentenses
- use space after change user, both [when use current space] and [when query "use space"] is okay. 

**While graph_spaces info will not be maintained if use [ when query] and change space to another space, which is now not usually used in tests.**